Tools/Scripts/webkitpy/browserperfdash/browserperfdash_runner.py

```diff
@@ -50,6 +51,7 @@ def parse_args():
     parser.add_argument('--local-copy', dest='localCopy', help='Path to a local copy of the benchmark (e.g. PerformanceTests/SunSpider/).')
     parser.add_argument('--count', dest='countOverride', type=int, help='Number of times to run the benchmark (e.g. 5).')
     parser.add_argument('--timeout', dest='timeoutOverride', type=int, help='Number of seconds to wait for the benchmark to finish (e.g. 600).')
+    parser.add_argument('--timestamp', dest='timestamp', type=int, help='Date of when the benchmark was run that will be sent to the performance dashboard server. Format is Unix timestamp (second since epoch). Optional. Defaults to now.')
```

Comment:
`Defaults to now` might be a bit confusing. It's more like server will use "now" when this field is not specified.

Reply:
Yes, will update the help comment to clarify that.